templates/docs/content-guidelines.md

```diff
@@ -0,0 +1,209 @@
+context:
+  title: Content Guidelines
+  description: Content guidelines for the Canonical design system
+---
+
+Open source software connects and uplifts, and our content does the same. With a diverse and global community of users, our language should be clear, consistent, and accessible.
+
+Follow this overall guidance for all UX copy:
+
+- Put the most important information first
+- Use short words and short sentences
+- Use sentence case
+- Avoid "please" and "thank you" – in UI copy, they can vary by culture
+- Use inclusive language – avoid gendered pronouns and ableist words such as "see" or "view"
+
+## Abbreviations
+
+Do not use e.g. or i.e. Instead, use ‘for example’ or ‘meaning’.
+
+## Alt text for images
+
+Alt text is a short description that replaces an image for users who don’t see the image. They may rely on screen readers or other assistive technologies. This is also good for SEO.
+
+### How screen readers work
+
+Screen readers follow the HTML reading order to read and interpret content for users.
+
+When an image conveys important information, alt text is how users who don’t see the images consume that information. When a screen reader gets to an image, it will read the alt text to describe the content or purpose. Screen readers ignore images with empty alt attributes.
+
+For complex images like graphs or diagrams, consider putting a longer description somewhere else on the page. For example, in a nearby paragraph or through ARIA attributes.
+
+### When to write alt text
+
+- Always write alt text when the image adds meaning.
+- Use an empty alt attribute (`alt=""`) if the image doesn’t add meaning. For example, the currency icons below are decorative and do not add meaning. [Learn more about decorative images](https://www.w3.org/WAI/tutorials/images/decorative/).
+
+### How to write alt text for images
+
+- Be concise, ideally less than 125 characters.
+- Describe the image accurately.
+- Add information the image conveys that is not in titles, link text, or paragraphs.
+
+**Instructions for certain types of images:**
+
+- **Card images:** Images inside cards or with links can be tricky. The screen reader will include the card title and link text, so alt text just needs to fill in any gaps. For example below, the card and link text tell the user a Dell XPS 13 Plus is a featured device, the missing information is it’s a laptop.
+
+<img src="https://assets.ubuntu.com/v1/1db0cd77-card-image-featured-device.png" alt="" width="250">
+
+- **Charts and diagrams:** If the chart is already explained in the text, use an empty alt attribute (`alt=""`). For example:
+
+<img src="https://assets.ubuntu.com/v1/0dd0ff70-empty-alt-attribute.png" alt="A screenshot of a page that describes the LTS support cycle of Ubuntu. A table summarizing the information is provided afterwards, with an empty alt-text." width="600">
+
+- **Graphs:** Describe the key trend or insight depicted in the graph so users understand the main data point. For example:
```

Comment:
We should at least remove the "for example:" at the end of the line so that reading the page does not feel strange - then flag that as a separate issue.

Reply:
@advl Leaving to you to flag that as a separate issue, resolving the conversation so this can get merged